string_manipulation/UnitTests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from ddt import ddt, data, unpack
+
```

Comment:
File names must not start with capital letters.
Violation: `UnitTests.py`
Correction: `unit_tests.py`